libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/NavigationViewModel.java

```diff
@@ -258,6 +260,7 @@ private void addMilestones(NavigationViewOptions options) {
   private ProgressChangeListener progressChangeListener = new ProgressChangeListener() {
     @Override
     public void onProgressChange(Location location, RouteProgress routeProgress) {
+      Timber.d("NAV_DEBUG - onProgressChange");
```

Comment:
I imagine this slipped in by mistake (debugging purposes). We should remove this log.